.github/workflows/pr-checks.yaml

```diff
@@ -89,15 +84,34 @@ jobs:
 
       - name: Setup-cli
         uses: ./.github/actions/setup-cli
+        with:
+          start-minikube: true
+
+      - name: Wait for push
+        uses: lewagon/wait-on-check-action@1b1630e169116b58a4b933d5ad7effc46d3d312d
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          check-name: "Push operator images (PR)"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 60
+
+      - name: Extract image references
+        uses: marcofaggian/action-download-multiple-artifacts@v3.0.8
+        with:
+          names: operator-refs-${{github.event.pull_request.number}}-${{github.event.pull_request.head.sha}}
 
       - name: Acceptance tests
         timeout-minutes: 60
         run: |
-          eval $(minikube docker-env)
-          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
+          export OPERATOR_INDEX_IMAGE_REF=$(cat operator-index.ref)
```

Comment:
if you go with the proposal from above, then all vars could be initialized via simple:

```
source ./operator.refs
```